src/Verify.Tests/ApplyScrubbersTests.cs

```diff
@@ -0,0 +1,25 @@
+// public class ApplyScrubbersTests
```

Comment:
why is this commented out?

Reply:
So, I should have deleted the file. The reason for this is that having these tests uncommented breaks other tests in random (because ApplyScrubbers is a static class with global state and changing it during tests break other tests).

Reply:
move it to a different test project so it runs in isolation